src/twisted/protocols/_smb/security_blob.py

```diff
@@ -0,0 +1,329 @@
+# -*- test-case-name: twisted.protocols._smb.tests -*-
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+"""Implement the "security buffer" an intricate sub-protocol
+used for authentication
+
+Blobs implemented as DER-encoded ASN.1 objects
+which in turn encapsulate NTLMv2 challenge-response authentication
+
+
+much of code adapted from Mike Teo's pysmb """
+
+from typing import Optional, cast
+
+from pyasn1.codec.der import decoder, encoder  # type: ignore
+from pyasn1.type import char, constraint, namedtype, namedval, tag, univ  # type: ignore
```

Comment:
This should be depending on https://pypi.org/project/types-pyasn1/, not `type: ignore`ing the imports